docs/outlook/sensitivity-label.md

```diff
@@ -16,33 +16,34 @@ You can use the Office JavaScript API to implement sensitivity label solutions i
+> [!NOTE]
+> Support for the sensitivity label feature was introduced in [requirement set 1.13](/javascript/api/requirement-sets/outlook/requirement-set-1.13/outlook-requirement-set-1.13). For information about client support for this feature, see [Supported clients and platforms](#supported-clients-and-platforms).
 
 ## Prerequisites
 
-To preview the sensitivity label feature in your add-in, you must have a Microsoft 365 E5 subscription. For access to a free developer sandbox that includes a renewable E5 subscription, join the [Microsoft 365 Developer Program](https://developer.microsoft.com/microsoft-365/dev-program#Subscription).
+To implement the sensitivity label feature in your add-in, you must have a Microsoft 365 E5 subscription. For access to a free developer sandbox that includes a renewable E5 subscription, join the [Microsoft 365 Developer Program](https://developer.microsoft.com/microsoft-365/dev-program#Subscription).
 
-You must also set up your preferred Outlook client as follows.
+### Preview the sensitivity label feature in Outlook on Mac
 
-- For Outlook on Windows, install Version 2302 (Build 16130.20020) or later. Then, join the [Microsoft 365 Insider program](https://insider.microsoft365.com/join/Windows) and select the **Beta Channel** option to access Office beta builds.
-- For Outlook on Mac, install Version 16.71.312.0 or later. Then, join the [Microsoft 365 Insider program](https://insider.microsoft365.com/join/Mac) and select the **Beta Channel** option to access Office beta builds.
-- For Outlook on the web, ensure that the **Targeted release** option is set up on your Microsoft 365 tenant. To learn more, see the "Targeted release" section of [Set up the Standard or Targeted release options](/microsoft-365/admin/manage/release-options-in-office-365#targeted-release).
+To preview the sensitivity label feature in Outlook on Mac, install Version 16.71.312.0 or later. Then, join the [Microsoft 365 Insider program](https://insider.microsoft365.com/join/Mac) and select the **Beta Channel** option to access Office beta builds.
 
 ### Supported clients and platforms
 
 The following table lists client-server combinations that support the use of the sensitivity label feature in Outlook add-ins. Excluded combinations aren't supported.
 
 |Client|Exchange Online|
 |------|------|
-|**Windows**<br>Version 2302 (Build 16130.20020) or later|Supported|
+|**Windows**<br>Version 2304 (Build 16313.10000) or later|Supported|
 |**Mac**<br>Version 16.71.312.0 or later|Supported|
 |**Web browser (modern UI)**|Supported|
 |**iOS**|Not applicable|
 |**Android**|Not applicable|
 
 ## Configure the manifest
 
+> [!NOTE]
+> The sensitivity label feature isn't yet supported for the [Teams manifest for Office Add-ins (preview)](../develop/json-manifest-overview.md).
```

Comment:
Should we keep this "Teams manifest" naming?

Reply:
https://review.learn.microsoft.com/en-us/office/dev/add-ins/develop/json-manifest-overview?branch=main

Reply:
Thanks for catching this! I missed this when I was resolving merge conflicts. Name here and in other articles have been corrected.